StatePrinter.Tests/IntegrationTests/ObjectGraphsTest.cs

```diff
@@ -173,37 +164,47 @@ public void MegaCyclicGraph_Curlybrace()
         Name = ""grandMom""
         Mother = null
         Children = new List<Human>()
-        Children[0] =  -> 0
+        {
+            -> 0
+        }
         Father = null
     }
     Children = new List<Human>()
-    Children[0] = new Human(), ref: 1
     {
-        Name = ""son""
-        Mother =  -> 0
-        Children = new List<Human>()
-        Father = new Human(), ref: 2
+        new Human(), ref: 1
         {
-            Name = ""grandDad""
-            Mother = null
+            Name = ""son""
+            Mother = -> 0
             Children = new List<Human>()
-            Children[0] =  -> 0
-            Children[1] =  -> 1
-            Children[2] = new Human(), ref: 3
             {
-                Name = ""daughter""
-                Mother =  -> 0
+            }
+            Father = new Human(), ref: 2
+            {
+                Name = ""grandDad""
+                Mother = null
                 Children = new List<Human>()
-                Father =  -> 2
+                {
+                    -> 0
```

Comment:
this is really hard to read - if it wasn't for the old output in the diff, I would not know it was the children.